common/src/leap/soledad/common/couch.py

```diff
@@ -476,9 +434,14 @@ def __init__(self, url, dbname, replica_uid=None, ensure_ddocs=True):
             self._set_replica_uid(replica_uid)
         if ensure_ddocs:
             self.ensure_ddocs_on_db()
-        # initialize a thread pool for parallelizing get_docs()
-        self._sem_pool = threading.BoundedSemaphore(
-            value=self.MAX_GET_DOCS_THREADS)
+        self._cache = None
```

Comment:
I'm still not really happy with the cache being setted from outside. But I'll accept the PR and hope for a future refactor.

Why does the couchdb code live in soledad.common anyway? it's only used in the server (and yes in leap.mx, but I'm ok with leap.mx depending on soledad.server).